src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -121,13 +122,30 @@ private void translateMessages(List<Message> messages, String targetLanguageCode
   
   //a helper function that takes a String value and returns a score
   private float getSentimentScore(String text) throws IOException {
-	Document doc = Document.newBuilder()
-      .setContent(text).setType(Type.PLAIN_TEXT).build();
-
-	LanguageServiceClient languageService = LanguageServiceClient.create();
-	Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
-	languageService.close();
-
-	return sentiment.getScore();
+	// Document doc = Document.newBuilder()
+      // .setContent(text).setType(Type.PLAIN_TEXT).build();
+
+	// LanguageServiceClient languageService = LanguageServiceClient.create();
+	// System.out.print(doc);
+	// AnalyzeSentimentResponse response = languageService.analyzeSentiment(doc);
+	// //Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+	// languageService.close();
+
```

Comment:
Do you still need these comments?

Reply:
Yes. Those commented are actually what I need, and those uncommented lines were copied from the official documentation. Both of them give an error when submitting a message